introduction.md

```diff
@@ -9,3 +9,14 @@ I am Jimmy Stevens. I am a 6 foot white male
 My favorite Code School path is JavaScript. I
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?